CONTRIBUTING.md

```diff
@@ -33,7 +33,7 @@ You can read a description of the [building and debugging process here].
 If you want simply to build the `jenkins.war` file as fast as possible without tests, run:
 
 ```sh
-mvn -am -pl war,bom -DskipTests -Dspotbugs.skip clean install
+mvn -am -pl war,bom -DskipTests -Dspotbugs.skip -Dspotless.check.skip clean install
```

Comment:
should probably just move to the quick-build profile?
https://github.com/jenkinsci/pom/blob/master/pom.xml#L899